### PR TITLE
Switch from Travis to Concourse for the scheduled runs

### PR DESCRIPTION
**Pull request by issyl0**
Travis has been broken recently, and the visibility of how this runs
wasn't very obvious. So I thought it could do with some improvements.

This switches to use Concourse to run the scheduled task as this is
the GDS-official blessed provider for these things. And it improves
the time at which the task runs so that people may have time to review
docs. ;-)

### CODE REVIEW

**Comment by issyl0** [comment]
This already applied to Concourse. 🤠

File: Rakefile [hunk 1]
```diff
@@ -4,21 +4,36 @@ require 'chronic'
 task default: ["notify:expired"]
 
 namespace :notify do
-  pages_url = ENV.fetch("SITE_PAGE_API_URL")
+  pages_urls = [
```
Comment (galund) [inline Rakefile:7]: For local testing ISTM you might want to be able to override this - having that in the environment might make that easier?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+on: [push, pull_request]
```
Comment (galund) [inline .github/workflows/ci.yml:1]: Does this have to be in a hidden directory? Seems likely to make it harder to discover/ grok what's going on.

Comment (issyl0) [reply .github/workflows/ci.yml:1]: It does, it's a [GitHub convention](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#about-yaml-syntax-for-workflows).

File: Rakefile (hunk 1 above)
Comment (issyl0) [reply Rakefile:7]: Equally you can just add another couple of lines to this array... I have no strong opinions: decoubling this from the matrix builds of Travis was my main aim and I didn't see how you could do that while keeping the individual envvar settings. Unless the envvar value is a comma-separated single string or something that we split on commas when we run the Rake task.

File: README.md [hunk 3]
```diff
@@ -15,32 +15,21 @@ It will look at the pages API for your site, find all pages that have expired, a
...
-[encrypt]: https://docs.travis-ci.com/user/encryption-keys/#usage
-[webhook]: https://api.slack.com/incoming-webhooks
+[Rakefile]: https://github.com/alphagov/tech-docs-monitor/blob/master/Rakefile
 
 ### General configuration
 
-If you want to use something other than Travis CI to deploy the notifier, you must make sure all its environment variables are defined:
+The following environment variables are necessary:
 
 * `SITE_PAGE_API_URL`: The full URL to your site's `/api/pages.json` file.
```
Comment (benthorner) [inline README.md:34]: I thought we [removed this one](https://github.com/alphagov/tech-docs-monitor/pull/19/commits/08d081f3b843bcd8be4656d99c60dce81fbbbed4)?

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+      - uses: actions/cache@v1
+        with:
+          path: vendor/bundle
+          key: bundle-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: bundle
+      - run: bundle install --jobs 4 --retry 3 --deployment
+      - run: bundle exec rspec
```
Comment (benthorner) [inline .github/workflows/ci.yml:14]: Should we also be doing [this](https://github.com/alphagov/tech-docs-monitor/pull/19/commits/05b00db7a860ea362342cb225c478c8409f6751a#diff-f9a62833538ab14fff849c2be16ab986L13)?

Comment (issyl0) [reply .github/workflows/ci.yml:14]: It is possible to run the app locally and see what it would send, so that should be fine for local development.

Comment (benthorner) [reply .github/workflows/ci.yml:14]: Sure, that makes sense. It's really an integration test in disguise, so if we need it, we could make it work that way instead.

**Pull request merged**